components/textbox/src/_mixin.scss

```diff
@@ -271,18 +271,33 @@ $utrecht-support-prince-xml: false !default;
   }
 }
 
+@mixin utrecht-textbox--aantalmarkten-size {
+  /* Nummer onder 9999 */
+  --utrecht-textbox-value-max-length: 10;
+}
+
 @mixin utrecht-textbox--postal-code-nl-size {
   /* Dutch postal code: 4 numbers, 1 space and 2 letters */
   --utrecht-textbox-value-max-length: 7;
 }
 
+@mixin utrecht-textbox--postbus-nummer-size {
+  /* A postbusnummer consists of at least one and at most five digits. */
+  --utrecht-textbox-value-max-length: 2;
+}
+
+@mixin utrecht-textbox--postcode-size {
+  /* A postcode consists of four numbers and two letters. */
+  --utrecht-textbox-value-max-length: 2;
```

Comment:
Waarom `2` als het 6 karakters en misschien een 7e voor de spatie kan bevatten? Bijvoorbeeld: `1234 AA`

Reply:
Ik heb de message bijgewerkt dat het 7 karakters kan bevatten, zoals aangegeven in het voorbeeld. Maar ik vind 7 max-lenght veel te groot. Misschien mis ik iets?

`--utrecht-textbox-value-max-length: 2;`

<img width="300" alt="Screenshot 2024-11-26 at 16 09 32" src="https://github.com/user-attachments/assets/a2deb48a-3095-4904-84ad-48ee849f3ba4">

`--utrecht-textbox-value-max-length: 7;`

<img width="300" alt="Screenshot 2024-11-26 at 16 09 49" src="https://github.com/user-attachments/assets/4d050c94-4398-4196-b591-2b818fcd4d2f">